readme.md

```diff
@@ -116,6 +116,7 @@
 - [Actions on Google](https://github.com/ravirupareliya/awesome-actions-on-google#readme) - Developer platform for Google Assistant.
 - [ESP](https://github.com/agucova/awesome-esp#readme) - Low-cost microcontrollers with WiFi and broad IoT applications.
 - [Deno](https://github.com/denolib/awesome-deno#readme) - A secure runtime for JavaScript and TypeScript that uses V8 and is built in Rust.
+- [Cloud Native](https://github.com/rootsongjc/awesome-cloud-native#readme) - A curated list of cloud native tools, software, and tutorials.
```

Comment:
This should describe what Cloud Native is, not the list itself. Please read all the guidelines.